mirrord/agent/src/outgoing.rs

```diff
@@ -190,7 +190,7 @@ impl TcpOutgoingApi {
                 }
                 else => {
                     // We have no more data coming from any of the remote hosts.
-                    warn!("interceptor_task -> no messages left");
+                    trace!("interceptor_task -> no messages left");
```

Comment:
Looks like it from the code but in reality the agent keeps logging this every second or so. @meowjesty Care to shed some light?

Reply:
This is an issue now because we're keeping the agent alive right? So now we get this due to not having any layer using this task?

I think changing the message is good, the proper way to stop having these looping logs (think we're going to find some more) is to have a better task cleanup mechanism, handling errors/closes.